### PR TITLE
[New] Add ProgressBar component

### PR DESCRIPTION
**Pull request by alphaoreo**
⭐ New Features:
- Added ProgressBar component
- Added tests
- Updated Code Coverage

### CODE REVIEW

**Comment by dynamic11** [comment]
@alphaoreo Prettier seems to be failing. Please check what file is causing the error and fix

**Comment by alphaoreo** [comment]
> @alphaoreo Prettier seems to be failing. Please check what file is causing the error and fix

I believe I found the file causing the problem and have fixed it.

File: src/components/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ProgressBarRB from 'react-bootstrap/ProgressBar';
+import '../../style.css';
+
+export interface ProgressBarProps {
+  /** Show label that represents visual percentage. EG. 60% */
+  label?: React.ReactNode;
+  /** Hide's the label visually. */
+  visuallyHidden?: boolean;
```
Comment (dynamic11) [inline src/components/ProgressBar/ProgressBar.tsx:9]: rename to isLabelHidden

File: src/components/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  ...rest
+}: ProgressBarProps) => (
+  <ProgressBarRB
+    label={label}
+    visuallyHidden={visuallyHidden}
+    max={max}
+    min={min}
+    now={now}
+    {...rest}
+    className={className}
```
Comment (dynamic11) [inline src/components/ProgressBar/ProgressBar.tsx:36]: move classname above rest

File: src/components/ProgressBar/ProgressBar.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  visuallyHidden = false,
+  max = 100,
+  min = 0,
+  now,
+  className,
+  ...rest
+}: ProgressBarProps) => (
+  <ProgressBarRB
+    label={label}
+    visuallyHidden={visuallyHidden}
```
Comment (dynamic11) [inline src/components/ProgressBar/ProgressBar.tsx:31]: update this variable name (see above)

**Review by dynamic11 — CHANGES_REQUESTED** [review]
Please see my requested changes and we need to write tests

**Review by dynamic11 — APPROVED** [review]
LGTM 🚀

**Pull request merged**